src/assets/style/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+//Mobile-first Media Queries
+
+//Breakpoints
+$tablet: (min-width: 768px);
```

Comment:
can we move these variables in to _variables file so that all variables are at one place? 
Also, rather than defining min-width in here, can we just have variables like - $media-breakpoint-tablet: 768px